ring_buffer/ring_buffer.py

```diff
@@ -1,11 +1,15 @@
 class RingBuffer:
-  def __init__(self, capacity):
-    self.capacity = capacity
-    self.current = 0
-    self.storage = [None]*capacity
+    def __init__(self, capacity):
+        self.capacity = capacity
+        self.current = 0
+        self.storage = [None]*capacity
 
-  def append(self, item):
-    pass
+    def append(self, item):
```

Comment:
Dang it, this is perfect. Love it. This is brilliant in its simplicity; take whatever is the current index number, overwrite whatever is in there, and then set the next self.current to where it needs to be. 

This is something I did not do; I immediately thought about data manipulation and complex math to get the data to where it needs to be; this is definitely where thinking simply and to the point is the way to go. Amazing job!